### PR TITLE
Create zh_cn.json and a small request

### PR DESCRIPTION
**Pull request by Imbarainbow**
Because of GFW, it's hard for we Chinese players to know mod in curseforge, so I'm wondering that if I can repost this mod to a Chinese forum called Mcbbs, I will make the download page link to curseforge pages and will not against any License.

### CODE REVIEW

File: src/main/resources/assets/slotlink/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    "block.slotlink.master":"网络主机",
+    "block.slotlink.master.tooltip":"存储链接的核心。\u00a75每个存储网络都需要一个。",
+    "block.slotlink.master.use1":"网络(%d, %d, %d):",
+    "block.slotlink.master.use2":"- 已链接的库存：%d",
+    "block.slotlink.master.use3":"- 总槽位数：%d",
+    "block.slotlink.master.use4":"- 最大可存储数量：%d",
+    "block.slotlink.request":"存储请求器",
+    "block.slotlink.request.hasNoMaster":"你需要将它与一个网络主机相链接。",
+    "block.slotlink.request.help0":"已链接槽位",
+    "block.slotlink.request.help1":"\u00a77Shift-单击：移动一组物品",
```
Comment (deirn) [inline src/main/resources/assets/slotlink/lang/zh_cn.json:12]: can you replace all `\u00a7` with `§`, please?

File: src/main/resources/assets/slotlink/lang/zh_cn.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    "item.slotlink.multi_dim_remote":"多维度远程存储访问器",
+    "item.slotlink.multi_dim_remote.tooltip":"可以访问任意距离，任意维度的存储网路。",
+    "item.slotlink.remote.hasNoMaster":"这个远程存储访问器未与任一存储网络相链接。",
+    "item.slotlink.remote.masterNotFound":"无法找到网络主机。",
+    "item.slotlink.remote.invalidDimension":"这个存储网络不在一个合法的维度中，请检查你的模组。",
+    "item.slotlink.remote.differentDimension":"你与该存储网络不在同一维度。",
+    "item.slotlink.remote.tooFarFromMaster":"你离网络主机太远了。",
+    "item.slotlink.remote.info":"与在%s，位置为(%d, %d, %d)的存储网络链接。",
+    "item.slotlink.remote.linked":"成功与在%s，位置为(%d, %d, %d)的存储网络链接。"
+}
```
Comment (deirn) [inline src/main/resources/assets/slotlink/lang/zh_cn.json:76]: also add a newline here

File: src/main/resources/assets/slotlink/lang/zh_cn.json (hunk 1 above)
Comment (deirn) [reply src/main/resources/assets/slotlink/lang/zh_cn.json:12]: no, remove all `\u00a7` and use `§` instead

**Comment by deirn** [comment]
cool

**Comment by deirn** [comment]
uploading 1.1.2 on curseforge, should be available soon

**Comment by Imbarainbow** [comment]
sure

**Pull request merged**